sql/core/src/main/scala/org/apache/spark/sql/streaming/StreamingQueryManager.scala

```diff
@@ -408,15 +408,15 @@ class StreamingQueryManager private[sql] (
       query.streamingQuery.start()
     } catch {
       case e: Throwable =>
-        unregisterTerminatedStream(query)
+//        unregisterTerminatedStream(query)
         throw e
     }
     query
   }
 
   /** Notify (by the StreamingQuery) that the query has been terminated */
   private[sql] def notifyQueryTermination(terminatedQuery: StreamingQuery): Unit = {
-    unregisterTerminatedStream(terminatedQuery)
+//    unregisterTerminatedStream(terminatedQuery)
```

Comment:
@WweiL is this by mistake? Should we revert this?

Reply:
oh shoot we should definitely revert this